Library/Homebrew/test/support/fixtures/cask/Casks/multiple-versions.rb

```diff
@@ -2,9 +2,10 @@
   arch arm: "arm", intel: "intel"
   platform = on_arch_conditional arm: "darwin-arm64", intel: "darwin"
 
-  version "1.2.3"
-  sha256 "67cdb8a02803ef37fdbf7e0be205863172e41a561ca446cd84f0d7ab35a99d94"
-
+  on_monterey :or_newer do
+    version "1.2.3"
+    sha256 "67cdb8a02803ef37fdbf7e0be205863172e41a561ca446cd84f0d7ab35a99d94"
+  end
```

Comment:
The order of these should be reversed, oldest to newest.

Reply:
Right, yes, that was the other PR's change to enforce that ordering. But the ordering makes sense to fix here from an example flow perspective. Lots of things to keep up with. 😅